docs/pages/setup/admin/labels.mdx

```diff
@@ -1,66 +1,274 @@
 ---
 title: Labels
-description: Labeling Nodes and Applications
+description: Labeling Teleport resources
 ---
 
-This guide explains how to label Teleport Nodes.
+Teleport allows you to add arbitrary key-value pairs to applications, servers,
+databases, and other resources in your cluster. You can use these to filter
+resources when making queries via `tctl` and `tsh` as well as to limit the
+resources that Teleport roles can access.
+
+There are two kinds of labels:
+
+- **Static labels** do not change over time while the `teleport` process is
+  running. Examples of static labels are the physical location of Nodes and the
+  name of the environment (e.g., `staging` vs. `production`).
+- **Dynamic labels**, also known as "label commands," allow you to generate
+  labels at runtime. Teleport will execute an external command on a Node at a
+  configurable frequency and the output of the command becomes the label value.
+  Examples include reporting load averages, the presence of a process, and the time after
+  the last reboot.
+
+This guide explains how to add labels to Teleport resources.
 
 ## Prerequisites
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
 (!docs/pages/includes/tctl.mdx!)
 
-## Labeling nodes and applications
+- A host where you will run a Teleport Node. In this guide, we will apply labels
+  to this Node.
 
-In addition to specifying a custom name for each Node, Teleport also allows for
-the application of arbitrary key-value pairs to each Node or app, called labels.
-There are two kinds of labels:
+<Admonition title="Resources you can label" type="tip">
 
-- **static labels** do not change over time while the `teleport` process is
-  running. Examples of static labels are the physical location of Nodes and the
-  name of the environment (e.g., staging vs. production).
-- **dynamic labels**, also known as "label commands," allow you to generate
-  labels at runtime. Teleport will execute an external command on a Node at a
-  configurable frequency and the output of the command becomes the label value.
-  Examples include reporting load averages, the presence of a process, and the time after
-  the last reboot.
+You can label any resource managed by Teleport using the instructions in this
+guide. See the following guides for how to add resources to a Teleport cluster:
```

Comment:
Labeling works differently for Windows desktops (there is no dynamic labels because there is no Teleport software running on the desktop). `docs/pages/desktop-access/rbac.mdx` has a labeling section that would be good to link to in the further reading section.

Databases are also a bit different when discovered automatically.